templates/_layout-brandstore.html

```diff
@@ -12,6 +12,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="{% block meta_description %}Snaps are containerised software packages that are simple to create and install. They auto-update and are safe to run. And because they bundle their dependencies, they work on all major Linux systems without modification.{% endblock %}">
+    <meta name="robots" content="noindex">
```

Comment:
Why removing indexing from brand store layout? Was it intentional?

Reply:
Right now there is no benefit indexing the brand stores we have. Surely once they are more established it will be more beneficial to do it so.